cylc/rose/entry_points.py

```diff
@@ -221,7 +224,6 @@ def record_cylc_install_options(
     rose_suite_conf = add_cylc_install_to_rose_conf_node_opts(
         rose_suite_conf, cli_config
     )
-    identify_templating_section(rose_suite_conf)
```

Comment:
Shouldn't we continue to check the merged config?

Reply:
Yes. I've put it back.